### PR TITLE
Change in utils/bam_handler.py to better handle reads with missing mates.

### PR DESCRIPTION
**Pull request by adrosenbaum**
Also Make Mutacc version available through CLI.

### CODE REVIEW

File: mutacc/utils/bam_handler.py [hunk 1]
```diff
@@ -100,16 +100,20 @@ def find_reads_from_region(self, chrom, start, end):
         for read in self.sam.fetch(chrom, start, end):
             #If name of read not among the keys in reads dict AND if both mates have not been found
             #allready make list to hold mates
```
Comment (henrikstranneheim) [inline mutacc/utils/bam_handler.py]: allready -> already

File: mutacc/utils/bam_handler.py [hunk 2]
```diff
@@ -100,16 +100,20 @@ def find_reads_from_region(self, chrom, start, end):
...
 
-                self.reads[read.query_name] = []
+            read_name = read.query_name
+
+            if read_name not in self.reads.keys() and \
+               read_name not in self.found_reads:
+
+                self.reads[read_name] = []
+
             #If both mates are not found allready, append read to mate list in reads
```
Comment (henrikstranneheim) [inline mutacc/utils/bam_handler.py]: already

File: mutacc/utils/bam_handler.py [hunk 3]
```diff
@@ -169,23 +153,34 @@ def find_reads_from_region(self, chrom, start, end):
...
-                )
 
 
     @property
     def record_number(self):
 
         return len(self.found_reads)
 
+    def find_mate(self, read):
+
```
Comment (henrikstranneheim) [inline mutacc/utils/bam_handler.py]: Could you add proper doc strings to all your defs in another PR

Comment (adrosenbaum) [reply mutacc/utils/bam_handler.py]: absolutely

**Pull request merged**